docs/Using-Fleet/CIS-Benchmarks.md

```diff
@@ -0,0 +1,77 @@
+
+The Center for Internet Security website offers documentation for all CIS Benchmarks in PDF format. Such PDFs document all the checks, their description, rationale and how to remediate them.
+
+Fleet has implemented CIS benchmarks for the following platforms:
+- CIS Apple macOS 13.0 Ventura Benchmark v1.0.0 - 11-14-2022 (82 checks) 
+- CIS Microsoft Windows 10 Enterprise Benchmark v1.12.0 - 02-15-2022 (Coming Soon!)
+
+## Manual vs Automated
+
+There are two types of CIS Benchmark checks, "Manual" and "Automated".
+- Automated: Represents recommendations for which assessment of a technical control can be fully automated and validated to a pass/fail state
+- Manual: Represents recommendations for which assessment of a technical control cannot be fully automated and requires all or some manual steps to validate that the configured state is set as expected.
+
+Fleet only implements "Automated" checks. "Manual" checks cannot be automated and/or require administrators to decide what's best for their organization's requirements.
+
+## Example CIS check
+
+Sample: `CIS - Ensure Install of macOS Updates Is Enabled` for macOS 13 Ventura Benchmark v1.0.0.
+
+<img src=https://user-images.githubusercontent.com/2073526/220428662-b324f6b7-7209-49a5-ae9f-4131316d8d1a.png />
+
+Such policy corresponds with the following entry in [cis-policy-queries.yml](https://raw.githubusercontent.com/fleetdm/fleet/main/ee/cis/macos-13/cis-policy-queries.yml):
+```yml
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: CIS - Ensure Install of macOS Updates Is Enabled (MDM Required)
+  platforms: macOS
+  platform: darwin
+  description: Ensure that macOS updates are installed after they are available from Apple.
+  resolution: "Ask your system administrator to deploy an MDM profile that enables automatic install of macOS updates."
+  query: SELECT 1 FROM managed_policies WHERE domain='com.apple.SoftwareUpdate' AND name='AutomaticallyInstallMacOSUpdates' AND value=1 LIMIT 1;
+  purpose: Informational
+  tags: compliance, CIS, CIS_Level1, CIS-macos-13-1.4
+  contributors: sharon-fdm
+```
+
+- The `description` briefly explains the check and its rationale.
+- The `resolution` explains how administrators can remediate a failing check on a host.
+- The `query` field is the osquery query that will execute on devices.
+
+## Requirements
+
+Following are the requirements to use the CIS Benchmarks in Fleet:
+
+- Fleet must be Premium licensed.
+- Devices must be running [Fleetd](https://fleetdm.com/docs/using-fleet/orbit), the osquery manager from Fleet. Fleetd can be built with [fleetctl](https://fleetdm.com/docs/using-fleet/adding-hosts#osquery-installer).
+- Devices must be enrolled to an MDM solution.
+- On macOS, the orbit executable in Fleetd must have "Full Disk Access", see [Grant Full Disk Access to Osquery on macOS](Adding-hosts.md#grant-full-disk-access-to-osquery-on-macos).
+
+## Apple macOS 13 Ventura Benchmark v1.0.0
```

Comment:
I think this section is better titled "How to add a benchmark"

Then I think we ought to have another section that's this title and then in it, we should list the CIS benchmarks which are manual.

Reply:
> I think this section is better titled "How to add a benchmark"

Makes sense!

> Then I think we ought to have another section that's this title and then in it, we should list the CIS benchmarks which are manual.

I don't follow this part. As in one by one list their title?

Reply:
Sorry, that dfeinitely wasn't clear.

I meant, there should be a general section called "how to add a benchmark" which describes in general, how to do it.

Then, there should be specific sections for each benchmark that notes ifs and buts. For example, In the specific section for the macos 13 benchmark, we should list which policies require manual checking. And any other notes.

Reply:
> Then, there should be specific sections for each benchmark that notes ifs and buts. For example, In the specific section for the macos 13 benchmark, we should list which policies require manual checking. And any other notes.

OK, so we would like to list here all 22 "Manual" checks (that are not part of the yml) and add a sample how a user would implement one of the manual checks via Policies? (if possible, some of the manual checks might be manual because there isn't an automated way at all).

Makes sense?

Reply:
Hmmmm... I think we should draw a sensible line as to what info to include and what to link out to. I scheduled a quick call for us tomorrow to go through it together.